gravis3d/body.py

```diff
@@ -29,15 +29,15 @@ def attract(self, attractor):
         #attractor.velocity -= force*dt
         #Sattractor.move()
         self.move()
-
     def is_collision(self, other):
-        if mag(self.pos-other.pos) <= mag(self.radius) + mag(other.radius) :
+        if (mag(self.pos-other.pos) <= self.radius + other.radius):
```

Comment:
We don't need `if ()` with brackets in python, that's what we do in c++

Reply:
That's what i do for my simplicity but here we cannot use mag over radius that's why i had changed it